ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -11,6 +11,31 @@
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 
 
+# sign
+@handle_frontend_test(
+    fn_tree="tensorflow.math.sign",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
```

Comment:
`tensorflow` also [supports](https://www.tensorflow.org/api_docs/python/tf/math/sign) `complex` dtypes as input to `tf.math.sign`. Kindly ensure the function gets tested for them too 🙂